src/components/BibleDropDown/BibleDropDown.md

```diff
@@ -1,10 +1,12 @@
 ```js
 import { useState } from 'react';
-import BibleDropDown from './BibleDropDown';
```

Comment:
As BibleBook  is the main component it can be renamed as BibleDropDown otherwise it will not show in the styleguide, the old BibleDropDown and BookContext can be deleted as you are no longer using context api. If you do rename it and delete the old files you can undo this line and the index.js export line also